helix-view/src/theme.rs

```diff
@@ -14,15 +14,18 @@ use toml::{map::Map, Value};
 use crate::graphics::UnderlineStyle;
 pub use crate::graphics::{Color, Modifier, Style};
 
+// store default theme in to byte array.
+const DEFAULT_THEME_DATA: &[u8] = include_bytes!("../../theme.toml");
+const BASE16_DEFAULT_THEME_DATA: &[u8] = include_bytes!("../../base16_theme.toml");
+
 pub static DEFAULT_THEME: Lazy<Theme> = Lazy::new(|| Theme {
     name: "default".into(),
-    ..toml::from_slice(include_bytes!("../../theme.toml")).expect("Failed to parse default theme")
+    ..toml::from_slice(DEFAULT_THEME_DATA).expect("Failed to parse default theme")
 });
 
 pub static BASE16_DEFAULT_THEME: Lazy<Theme> = Lazy::new(|| Theme {
     name: "base16_theme".into(),
```

Comment:
A bit unrelated to this PR but this should be `"base16_default"`. There's a mismatch where you can choose this theme with `:theme base16_default` but then `:theme` (which prints the current theme) says "base16_theme".